src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.cs

```diff
@@ -80,7 +80,7 @@ public void Initialize()
 
                 VSColorTheme.ThemeChanged += VSColorTheme_ThemeChanged;
 
-                QueueColorSchemeUpdate(themeChanged: true);
+                UpdateColorScheme(themeChanged: true);
```

Comment:
can we comment what this is even for in the first place?  why do we need to update anything as our package loads?  don't we already know the colors?  and why are we saying "themeChanged: true"?

Basically, i don't know what this code is for, i don't know why it's correct, and i don't know what would happen if someone just removed it in the future.

Reply:
Update with a comment